src/workerd/api/tests/actor-stub-test.js

```diff
@@ -13,7 +13,7 @@ export class DurableObjectExample {
     return new Response("OK");
   }
 
-  foo() {
+  async foo() {
```

Comment:
Obviously there's lots of tests still to implement here but make sure to include some tests that ensure that types that can't be v8 serialized (e.g. `Symbol`, `SharedArrayBuffer`, etc).

Reply:
The test in my internal PR verifies we throw an error:
- if the client calls an rpc method and passes arguments that cannot be serialized, and
- if the server tries to return a value that cannot be serialized.

There's some more discussion about potential issues on the server side [in this comment thread](https://github.com/cloudflare/workerd/pull/1028#discussion_r1416425364)